Tutorial_1c_Line_Ratios.ipynb

```diff
@@ -0,0 +1,736 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    cube_12co_convolved = cube_12co.convolve_to(beam)
</pre><p>Similarly, you should be able to simplify these lines to <code>cube_12co.convolve_to(cube_13co.beam)</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Smithsonian/sma-science-tutorials/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>